string name # name of landmark
float64 distance # distance to the landmark, in meters
